<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_interface</class>
 <widget class="QDialog" name="main_interface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>494</width>
    <height>362</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>R-SODIUM Ultra SSD Enclosure Manager</string>
  </property>
  <property name="windowIcon">
   <iconset resource="usm.qrc">
    <normaloff>:/img/icon.ico</normaloff>:/img/icon.ico</iconset>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>151</width>
     <height>91</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>141</width>
     <height>70</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="usm.qrc">:/img/logo.svg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="overview_button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Overview</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SATA_Interface_button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>SATA Config</string>
   </property>
  </widget>
  <widget class="QPushButton" name="boxmode_button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Box Mode</string>
   </property>
  </widget>
  <widget class="QPushButton" name="about_button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="sataconfig">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>311</width>
     <height>341</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <italic>false</italic>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="title">
    <string/>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <widget class="QCheckBox" name="JBOD">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>136</y>
      <width>111</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>SPAN / JBOD</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="R1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>106</y>
      <width>80</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>RAID 1</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="R0">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>76</y>
      <width>80</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>RAID 0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>26</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="text">
     <string>ASM1352R RAID Config</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QCheckBox" name="PM">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>166</y>
      <width>111</width>
      <height>19</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Normal</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="sata_execute">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>280</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Execute</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="powertime">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>216</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>215</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="text">
     <string>SATA power-up after</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>220</y>
      <width>53</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>second/s.</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="boxmode">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>9</y>
     <width>311</width>
     <height>341</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Enclosure Mode</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QCheckBox" name="combinemode">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Combine Mode (NVMe + SATA)</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="nvmeonly">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>NVMe Only</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="sataonly">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>140</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>SATA Only</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>170</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Other Configuration</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QCheckBox" name="physicalenable">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>271</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Physical switch can still change modes.</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="storageconfig">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>261</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Keep mode config for next power up.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boxmode_execute">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>280</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Execute</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="overview">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>9</y>
     <width>311</width>
     <height>341</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="about">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>311</width>
     <height>341</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
  </widget>
  <zorder>sataconfig</zorder>
  <zorder>overview</zorder>
  <zorder>about</zorder>
  <zorder>overview_button</zorder>
  <zorder>label_2</zorder>
  <zorder>SATA_Interface_button</zorder>
  <zorder>boxmode_button</zorder>
  <zorder>about_button</zorder>
  <zorder>boxmode</zorder>
 </widget>
 <resources>
  <include location="usm.qrc"/>
 </resources>
 <connections/>
</ui>
